<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeGraphicsItemPropertiesWidget</class>
 <widget class="QWidget" name="ShapeGraphicsItemPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Trait</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Épaisseur</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Couleur</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KColorButton" name="m_color_kpb"/>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="m_size_dsb">
        <property name="maximumSize">
         <size>
          <width>16777212</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.200000000000000</double>
        </property>
        <property name="maximum">
         <double>50.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="m_style_cb">
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tiret</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pointillé</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Traits et points</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Traits points points</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tiret custom</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_filling_gb">
     <property name="title">
      <string>Remplissage</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="m_brush_style_cb">
        <item>
         <property name="text">
          <string>Aucun</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Plein</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Densité 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Densité 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Densité 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Densité 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Densité 5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Densité 6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Densité 7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horizontal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vertical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Croix</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diagonal arrière</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diagonal avant</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diagonal en croix</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Style</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Couleur</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KColorButton" name="m_brush_color_kpb"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_lock_pos_cb">
     <property name="text">
      <string>Verrouiller la position</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_close_polygon">
     <property name="text">
      <string>Polygone fermé</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_style_cb</tabstop>
  <tabstop>m_size_dsb</tabstop>
  <tabstop>m_color_kpb</tabstop>
  <tabstop>m_brush_style_cb</tabstop>
  <tabstop>m_brush_color_kpb</tabstop>
  <tabstop>m_lock_pos_cb</tabstop>
  <tabstop>m_close_polygon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
